action.yaml

```diff
@@ -21,6 +25,10 @@ inputs:
   cloud-run-locally:
     description: 'If true, run tests locally instead and upload results to Grafana Cloud k6'
     required: false
+  comment-cloud-test-run-url-on-pr:
```

Comment:
What happens if this runs on a non-PR? Shall it comment on the commit instead (Example: https://github.com/grafana/k6-infrastructure/commit/27f662a895e300b47672b42732b5410e2822a2c3#comments), or do nothing?

Also the naming looks a bit too long, I wonder if something like `cloud-run-link` could do the trick here :thinking: 

cc @dgzlopes since this is public/product interface

Reply:
> What happens if this runs on a non-PR? Shall it comment on the commit instead 

It might become too noisy to comment on commits so added it only for PRs, currently it skips commenting if the PR does not exist. 

> Also the naming looks a bit too long, I wonder if something like cloud-run-link could do the trick here 🤔

I agree the name of the input is little long, but I thought it makes more sense to be more explicit on what that particular input does.

Reply:
Updatd the action input name